python/lsst/ts/xml/generate_csv_table.py

```diff
@@ -27,7 +27,6 @@
 
 
 def generate_csv_table():
```

Comment:
Any reason for changing this file here?

Reply:
It wasn't passing black and flake8.

Reply:
I think it got caught in the middle of a black 22 -> 23 upgrade. We are holding back on that for now though so the original version should pass.

Reply:
Ok, hopefully that fixed it.